common/api/core-geometry.api.md

```diff
@@ -4526,7 +4526,6 @@ export class PolyfaceBuilder extends NullGeometryHandler {
     addTrianglesInUncheckedConvexPolygon(ls: LineString3d, toggle: boolean): void;
     addTriangulatedRegion(region: AnyRegion): void;
     addUVGridBody(surface: UVSurface, numU: number, numV: number, uMap?: Segment1d, vMap?: Segment1d): void;
-    applyStrokeCountsToCurvePrimitives(data: AnyCurve | GeometryQuery): void;
```

Comment:
I don't expect the deprecated method to be removed from this file.  Other deprecated methods remain, but extract-api precedes them with `// @deprecated`.  Can you try running extract-api again?

Reply:
Done